README.md

```diff
@@ -1,2 +1,45 @@
 # parse-hamnosys
 Sign language HamNoSys notation parsing tool.
+
+## Usage
+```
+$ python3 parse-hamnosys.py -sf <source_file> -df <destination_file>
+```
+
+## Usage example
+```
+$ python3 parse-hamnosys.py -sf hamnosys_example.txt -df hamnosys_parsed.txt
```

Comment:
minor issue: for venv python3 is specified, so after creating virtual env for `python` it should be default to call `python3`

Reply:
done :)